src/main/java/com/cloudbees/jenkins/plugins/bitbucket/client/BitbucketCloudApiClient.java

```diff
@@ -667,6 +678,47 @@ public BitbucketTeam getTeam() throws IOException, InterruptedException {
             return null;
         }
     }
+    /**
```

Comment:
Missing white space separation
```suggestion

    /**
```